Add my information about Git.

lisnyak-o opened the pull request.

miroslav-kolomiets reviewed (changes requested):
Looks great! But you make some changes in bottom of the file, maybe add a new line, you should not change other people code ;)
<img width="1335" alt="Снимок экрана 2020-10-13 в 8 48 15 PM" src="https://user-images.githubusercontent.com/14906260/95897130-b1d8dd80-0d95-11eb-9360-d331b0f899ca.png">

lisnyak-o commented:
Thanks @miroslav-kolomiets , I decided conflict, where`re maked mistakes.

miroslav-kolomiets reviewed (changes requested):
Now it's good, but you need to resolve conflict in your branch.

lisnyak-o commented:
I merge base branch and pull new changes in this project for my fork.

Temu4 commented on README.md:
```diff
@@ -63,6 +75,7 @@ PS. Check the Kottans in wheeeeeaaatt.
 
 
+
 # Hi Everyone =)
```
Please try to not change other people's code.

Temu4 commented on README.md:
```diff
@@ -1423,7 +1436,6 @@ I'm Kyrylo, I have always dreamed of programming and this a great opportunity fo
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
 ![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
----
```
As well as here ⬆️

lisnyak-o commented:
Guys, please see it`s right made ?

It was : 
![123](https://user-images.githubusercontent.com/20680156/96975977-eaaa4c80-1523-11eb-9d87-af8ff66ef4a0.png)
Result : 
![321](https://user-images.githubusercontent.com/20680156/96975981-ec741000-1523-11eb-9bb2-da245fa561a2.png)

miroslav-kolomiets commented on README.md:
```diff
@@ -46,6 +46,19 @@ It was an explosive start to front-end learning! At first it was hard for me bec
 
 ---
 
+
+
+# Hi Guys !
+My name is [Oleg](https://github.com/lisnyak-o). So, meet my cat, called Richard ;)
+
+![richard](gif/richard.jpg "Richard")
+
+If you’ve questions about the tasks or something you can always contact me at :
+# [linkendIn](https://www.linkedin.com/in/oleg-lisnyak-a888a017a/) or [GitHub](https://github.com/lisnyak-o)
+## You’re welcome !
+Version Control with GIT - Completed !
+# Let’s start to next step!
+
```
Simply, the PR should contain nothing but these lines.

miroslav-kolomiets commented on README.md:
```diff
@@ -56,6 +69,7 @@ It was an explosive start to front-end learning! At first it was hard for me bec
 I'm [Max](https://github.com/maxpetruk) and this is Greyjoy, the kitten I'm hoping to adopt soon :smile:![my_kitten](gif/my_kitten.jpg)
 
 
+
```
It shouldn't be.

miroslav-kolomiets commented on README.md:
```diff
@@ -72,6 +86,7 @@ PS. Check the Kottans in wheeeeeaaatt.
 
 
 
+
```
It shouldn't be too.

miroslav-kolomiets commented on README.md:
```diff
@@ -1444,7 +1459,6 @@ I'm Kyrylo, I have always dreamed of programming and this a great opportunity fo
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
 ![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
----
```
It shouldn't be too.

babayK0 commented:
@lisnyak-o resolve conflicts please)

lisnyak-o commented:
I leave empty line to the end for comment, but when I push my file it's line haven't, why  it's happend?

![Screenshot_1](https://user-images.githubusercontent.com/20680156/97561065-41ae9680-19e8-11eb-891f-c0794a22113d.png)
![Screenshot_2](https://user-images.githubusercontent.com/20680156/97561070-43785a00-19e8-11eb-8018-8ebea5529e6b.png)

Temu4 commented:
Hey @lisnyak-o !
I still see a lot of code changes with you commits.

I hope you can do smth like these: 
1. Turn off auto-format settings in your code redactor/IDE.
2. Roll back to initial state of this branch (before your changes)
3. Pull the latest changes from origin repo
4. Add you code

If you need more info, please, fill free to touch me 😸

lisnyak-o commented:
I hope everything worked out.
  What I did ?
1) git reset --hard HEAD
2) git fetch upstream
3) Added my own text at the end of the file, added a photo, then did => commit
4) git pull master
5) Solved the conflict. Accepted the current changes, and canceled the incoming (from the pull master).
6) git push

![image](https://user-images.githubusercontent.com/20680156/98292884-1f071980-1fb6-11eb-98f2-a6dd99869b2f.png)
Why this line changed to the same line I don't know. I didn't touch her!

lisnyak-o commented:
I'm already confused. Can I create a new PR? After all, I can't push a rollback via reset to the server.
And how then to solve this situation, I can not even imagine.

lisnyak-o commented:
I created a new PR.
